Havtorn/Source/Engine/Graphics/RenderManager.cpp

```diff
@@ -141,6 +87,40 @@ namespace Havtorn
 		InitEditorResources();
 		LoadDemoSceneResources();
 
+		RenderFunctions[ERenderCommandType::ShadowAtlasPrePassDirectional] = std::bind(&CRenderManager::ShadowAtlasPrePassDirectional, this, std::placeholders::_1);
```

Comment:
Based on the style in this method before this line, binding these RenderFunctions could be moved into its own method?
```suggestion
BindRenderActions();
```